src/theme/interaction.rs

```diff
@@ -34,6 +34,8 @@ pub struct InteractionPalette {
 /// button press.
 #[derive(Component, Debug, Reflect, Deref, DerefMut)]
 #[reflect(Component, from_reflect = false)]
+// Note that the ugly reflect attributes are due to
```

Comment:
I agree that it's ugly here, but at the same time, proc macro attributes can provide some great features as well. The current wording may suggest that users should avoid `#[reflect]` attributes in general.

Same comment for the other place this was written.

Reply:
[`d6d4000` (#241)](https://github.com/TheBevyFlock/bevy_quickstart/pull/241/commits/d6d40008734a566dda6ce4d1cf351d005b0dea11)